serengil/deepface
# Feature/custom output face from extract faces

### What has been done

Changed extract_faces parameters to be possible to return face image (croped) as rgb, bgr or grayscale and flag for normalized. By default, all parameters will return the same as before (RGB and normalized).

## How to test
Change parameters as desired:
```
faces = DeepFace.extract_faces(
    img_path = 'image.jpg',
    normalize_face=True,  # True or False
    color_face='bgr'  # rgb, bgr or gray
)
```
After extracting face can save to see the result:
```
Image.fromarray((faces[0]['face']*255).astype(np.uint8)).save('img.png')  # normalize_face == True
Image.fromarray((faces[0]['face']).astype(np.uint8)).save('img.png')  # normalize_face == False
```
